metricflow/cli/main.py

```diff
@@ -62,9 +67,32 @@ def cli(cfg: CLIContext, verbose: bool) -> None:  # noqa: D
         )
 
         click.echo(
-            f"💡 Please update to version {result.available_version}, released {result.release_date} by running:\n\t$ pip install --upgrade {PACKAGE_NAME}",
+            f"💡 Please update to version {result.available_version}, released {result.release_date} by running:\n"
+            f"\t$ pip install --upgrade {PACKAGE_NAME}",
         )
 
+    # Cancel queries submitted to the DW if the user precess CTRL + c / process is terminated.
+    # Note: docs unclear on the type for the 'frame' argument.
+    def exit_signal_handler(signal_type: int, frame) -> None:  # type: ignore
+        if signal_type == signal.SIGINT:
+            click.echo("Got SIGINT")
+        elif signal_type == signal.SIGTERM:
+            click.echo("Got SIGTERM")
+            signal.valid_signals()
```

Comment:
do we need this? checked the docs it says 
```
Return the set of valid signal numbers on this platform. 
This can be less than range(1, NSIG) if some signals are reserved by the system for internal use
```
doesn't seem like we're using the return value

Reply:
Will remove - this was left from testing.